.composer/composer.json

```diff
@@ -1,6 +0,0 @@
-{
-    "require": {
-        "php": ">=5.4",
```

Comment:
@aleeks, this line is still available in https://github.com/PrestaShop/PrestaShop/blob/develop/composer.json#L6